### PR TITLE
Added HTML to list of accepted filetypes

### LINKED ISSUE #11: Add support for compressing html files
Javascript and CSS are a good start, but it would be helpful to be able to compress html files as well, as they can get large in complex apps (especially if they have data baked in).

### PR DESCRIPTION
**Pull request by AABoyles**
Resolve #11

### CODE REVIEW

**Comment by gios** [comment]
@AABoyles Hi, thanks for your changes. Could you fix the tests and add a couple of files to `folder_to_compress` folder with for example .png, .html or .json extensions. Also, increase the version in package.json and package.lock.

**Comment by AABoyles** [comment]
Will do, thanks!

File: Gzipper.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ const lstat = util.promisify(fs.lstat)
 const readdir = util.promisify(fs.readdir)
 const mkdir = util.promisify(fs.mkdir)
 
+const validExtensions = [
```
Comment (gios) [inline Gzipper.js]: Use uppercase letters with underscore divider when declaring constants like OUTPUT_FILE_FORMAT_REGEXP.

**Comment by gios** [comment]
@AABoyles Thank you for your contribution, a new version has been published [v2.5.2](https://www.npmjs.com/package/gzipper)

**Pull request merged**